doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
+
+.. _migration_2.6:
+
+Migration guide for |NCS| v2.6.0 (Working draft)
+################################################
+
+This document describes the changes required or recommended when migrating your application from |NCS| v2.5.0 to |NCS| v2.6.0.
+
+Required changes
+****************
+
+The following changes are mandatory to make your application work in the same way as in previous releases.
+
+* For the Serial LTE Modem (SLM) application:
+
+  * The Zephyr settings backend has been changed from :ref:`FCB <fcb_api>` to :ref:`NVS <nvs_api>`.
+    Due to this one setting is lost after the switch if you are using the :ref:`liblwm2m_carrier_readme` library.
```

Comment:
Hmm, in that case it could probably be better with:
```suggestion
    Because of this, one setting is restored to its default value after the switch if you are using the :ref:`liblwm2m_carrier_readme` library.
```